### DISCUSSION TITLE
Moving fulcrum database to other internal drive

### DISCUSSION THREAD

**Discussion by Knrrm** [cculianu/Fulcrum#246] [Q&A] [answered]
My OS is Linux mint. My main drive is slowly but surely running out of space. In order to keep running my node for a while longer before buying new hardware and starting from scratch, I need to migrate fulcrum database to the other internal drive I have on my computer. If this is even possible, which are the exact steps I chronologically have to do to accomplish this?

**Comment by cculianu** [comment, marked as answer]
`cp -fpvra` is your friend here.

Also you can move individual subdirs or **top level** files from the data/ directory to a different drive .. and symlink them back in from the main folder... don't move the rocksdb table files within the subfolders tho or you will have a bad time...

**Reply by Knrrm** [reply]
Ok, thx a lot. Had to read up what all those flags do.  😅

I'm no expert unfortunately, certainly not with symlinks. Don't I have to change
`datadir = /path/to/a/dir`
in the conf file to point to the copied database folder? And I'm guessing I should shutdown fulcrum (and probably also bitcoin core?) before I start all this, right?

**Comment by cculianu** [comment]
Yes to both those things

**Reply by Knrrm** [reply]
Ok thx, is there anything else I need to consider to get this right?
